spaces/domains.json

```
@@ -697,5 +698,5 @@
   "vote.rei.network": "rei-network.eth",
   "vote.blockchaincuties.com": "cutiesofficial.eth",
   "vote.nuddybits.com": "nuddybits.eth",
-  "vote.dragon.xyz": "dragonmeta.eth"
+  "vote.dragon.xyz": "dragonmeta.eth",
```

@samuveth Do we need a comma at the end of the last line?
```suggestion
  "vote.dragon.xyz": "dragonmeta.eth",
```

---

Nope